db/schema.rb

```diff
@@ -51,7 +52,7 @@
     t.string   "token"
     t.string   "profile_picture"
     t.string   "profile_url"
-    t.string   "email"
+    t.string   "profile_email"
```

Comment:
Hmmm... the `migration` that causes this change doesn't seem to be included? :open_mouth: